Makefile

```diff
@@ -8,5 +8,5 @@ directories:
 	mkdir -p $(BIN_DIR)
 

```

Comment:
Please update the unit test makefile and ensure all tests work with these new changes

Reply:
test/Makefile  is updated.